File: FlowerObjc/FlowerObjc/Example/TextViewVC.m
//
//  TextViewVC.m
//  SGFastfishExample
//
//  Created by kingsic on 2020/12/22.
//  Copyright © 2020 kingsic. All rights reserved.
//

#import "TextViewVC.h"
#import "SGTextView.h"

@interface TextViewVC ()

@end

@implementation TextViewVC

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    self.navigationItem.title = @"SGTextView";
    self.view.backgroundColor = [UIColor whiteColor];

    SGTextView *tv = [[SGTextView alloc] init];
    tv.frame = CGRectMake(50, 100, self.view.frame.size.width - 100, 100);
    tv.backgroundColor = [UIColor greenColor];
    tv.layer.cornerRadius = 10;
    tv.layer.borderWidth = 1;
    tv.layer.borderColor = [UIColor blackColor].CGColor;
    tv.placeholder = @"分享美好事物……";
    tv.placeholderColor = [UIColor redColor];
    [self.view addSubview:tv];
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: Flower/Flower/Example/TextViewVC.swift
//
//  TextViewVC.swift
//  Flower
//
//  Created by kingsic on 2021/9/8.
//

import UIKit

class TextViewVC: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        view.backgroundColor = .white
        navigationItem.title = "SGTextView"
        // Do any additional setup after loading the view.
                
        let textView = SGTextView()
        textView.frame = CGRect.init(x: 20, y: navBarHeight + 20, width: screenWidth - 40, height: 200)
        textView.backgroundColor = .green
        textView.placeHolder = "请输入你想要表达的内容"
        textView.placeHolderColor = .red
        textView.limitNumber = 7
        view.addSubview(textView)
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destination.
        // Pass the selected object to the new view controller.
    }
    */

}
